climate.py

```diff
@@ -248,6 +253,7 @@ def update(self):
 
             try:
                 state = self._ac.poll()
+                power = self._ac.get_power()
```

Comment:
Not all devices can report power, unfortunately—mine can't, for instance. When I try to use this, the API raises an `InvalidRequestError`. Can we please catch that error (just for this line, not for the `poll`)?

Even better if we remember the firs time we hit this and don't keep trying every time.